lizmap/modules/lizmap/lib/Project/Qgis/Layer/EditWidget/ValueRelationConfig.php

```diff
@@ -79,6 +90,10 @@ protected function set(array $data): void
         if (array_key_exists('UseCompleter', $data)) {
             $data['UseCompleter'] = filter_var($data['UseCompleter'], FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE);
         }
+
```

Comment:
Could it be removed?

Reply:
Yes, it was late yesterday in the train.